subways/structure/route.py

```diff
@@ -319,46 +345,51 @@ def calculate_distances(self) -> None:
 
     def process_tags(self, master: OsmElementT) -> None:
         relation = self.element
+        tags = relation["tags"]
         master_tags = {} if not master else master["tags"]
-        if "ref" not in relation["tags"] and "ref" not in master_tags:
+        if "ref" not in tags and "ref" not in master_tags:
             self.city.notice("Missing ref on a route", relation)
-        self.ref = relation["tags"].get(
-            "ref", master_tags.get("ref", relation["tags"].get("name", None))
+        self.ref = tags.get(
+            "ref", master_tags.get("ref", tags.get("name", None))
         )
-        self.name = relation["tags"].get("name", None)
-        self.mode = relation["tags"]["route"]
+        self.name = tags.get("name", None)
+        self.mode = tags["route"]
         if (
-            "colour" not in relation["tags"]
+            "colour" not in tags
             and "colour" not in master_tags
             and self.mode != "tram"
         ):
             self.city.notice("Missing colour on a route", relation)
         try:
             self.colour = normalize_colour(
-                relation["tags"].get("colour", master_tags.get("colour", None))
+                tags.get("colour", master_tags.get("colour", None))
             )
         except ValueError as e:
             self.colour = None
             self.city.warn(str(e), relation)
         try:
             self.infill = normalize_colour(
-                relation["tags"].get(
+                tags.get(
                     "colour:infill", master_tags.get("colour:infill", None)
```

Comment:
```suggestion
                    "colour:infill", master_tags.get("colour:infill")
```

Reply:
Dozens occurrencies through the project. Let's consider it a programmer's style.